setup/emc-update-impl.bash

```diff
@@ -65,11 +65,14 @@ function _git_clone_or_update
 # Install ROS
 if [ ! -d /opt/ros/"$EMC_ROS_DISTRO" ]
 then
-    sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
+    if [ ! -f /etc/apt/sources.list.d/ros-latest.list ]
+    then
+        sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
 
-    wget http://packages.ros.org/ros.key -O - | sudo apt-key add -
+        curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | sudo apt-key add -
```

Comment:
For future reference: 

Conforms to the way it is described in the official documentation: for example here
http://wiki.ros.org/noetic/Installation/Ubuntu